### DISCUSSION TITLE
Doesn't show previous thrown exception

### DISCUSSION THREAD

**Discussion by DarkGhostHunter** [spatie/laravel-ignition#62] [Ideas]
Title.

Once an exception is catched, and contains a previously thrown exception, there is no UI or log for that previous exception.

The only way to retrieve it is manually editing the code.

**Comment by freekmurze** [comment]
We will add this in the near future.

**Comment by rasmuscnielsen** [comment]
Was this ever implemented? Just came across same issue. Would be really helpful indeed 🙏
